Libraries/Components/WebView/WebView.js

```diff
@@ -0,0 +1,416 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule WebView
```

Comment:
@narek-h Do we really need WebView.js since packager eats only WebView.desktop.js finally?

Reply:
@MaxRis , you are right. I left that somehow. Will be removed.